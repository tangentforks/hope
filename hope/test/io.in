uses list, lines;
lines(read "../test/hope_tut.in")@200;
write "Hello world\n";
