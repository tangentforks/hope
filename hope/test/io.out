>> "             beta -> beta;" : list char
Hello world
